packages/text-field/src/TextFieldPresenter.js

```diff
@@ -80,6 +80,10 @@ export default class TextFieldPresenter extends Component {
      * When true, causes the clear button to appear
      */
     showClearButton: PropTypes.bool,
+    /**
+     * Allow the field to stay focused persistently.
+     */
+    stayFocused: PropTypes.bool,
```

Comment:
While this will solve the problem, this prop's use case is a bit too specific. For example, to make the `TextFieldPresenter` always appear blurred, we would need to also add a `stayBlurred` prop.

The reason I suggested `props.focused` to be added was to be able to handle multiple use cases, while keeping the API ubiquitous and succinct. For more context, the blurred use case will be used in the upcoming multi-select dropdown feature.

If there's concern is about confusion between `state.focused` and `props.focused`, I'd rename `state.focused` to `state.hasFocus` for more clarity.

* `props.focused` — Determines whether the text field appears focused.
    - If `true` the text field appears focused
    - If `false` the text field appears blurred
    - If `undefined` fallback to `state.focused`
* `state.focused` (or rename `state.hasFocus`) — Whether the text field input is actually focused